ontobio/ontol.py

```diff
@@ -358,6 +358,73 @@ def node_type(self, id):
         """
         return self.node(id)['type']
 
+    def get_ancestors(self, go_term):
+        """
+        Returns the ancestors from the is_a, part_of relation traversed GO subontology of go_term's ancestors.
+
+        subontology() primarily used here for speed when specifying relations to traverse.
+        """
+        ### BFO:0000050 = part of
+        ### I assume "subClassOf" = is a?
+        all_ancestors = self.ancestors(go_term)
```

Comment:
@dustine32 

If you call `self.ancestors(go_term, reflexive=True)` then you won't have to do `all_ancestors.append(go_term)` as the term itself is included in the list of ancestors returned by `self.ancestors()`.

Reply:
Thanks @deepakunni3 ! I'll try that option out.

Reply:
Yep, it worked!